Resources/views/CRUD/list_array_recursion.html.twig

```diff
@@ -0,0 +1,17 @@
+{#
+
+This file is part of the Sonata package.
+
+(c) Thomas Rabaix <thomas.rabaix@sonata-project.org>, Toni Cornelissen <toni@iboss.nl>
+
+For the full copyright and license information, please view the LICENSE
+file that was distributed with this source code.
+
+#}
+    {% for key, val in value %}
```

Comment:
I've lifted most of that code from list_array.html.twig and kept the same indention. I've now de-indented the code so the first line of code starts at the first column.